src/org/loklak/data/ElasticsearchClient.java

```diff
@@ -943,4 +946,46 @@ public Query(final String indexName, final QueryBuilder queryBuilder, String ord
         }
         return list;
     }
+
+    public HashMap<String, HashMap<String, Long>> classifierScore(String index, String classifierName, ArrayList<String> classes) {
```

Comment:
Can the declaration be broken down into multiple lines like one parameter per line, as it exceeds 100 chars per line?